docs/changelog/107657.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 107312
```

Comment:
Thank you for your pull request. Before we can merge this, can you remove this changelog entry? The change you applied to x-pack/plugin/ml/build.gradle has no broader effect on build tools general functionality. therefore we do not want a changelog item. those changelogs are used to generate release notes for elasticsearch releases. 
this change is an internal change and not exposed to customers